functions.lisp

```diff
@@ -171,16 +171,15 @@ For generic implementation and getails, see `jrem'."
   (:method ((object (eql t))) object)
   (:method ((object null)) object)
   (:method ((object string)) object)
-  (:method ((object sequence))
+  (:method ((object vector))
     (map 'vector #'jcopy object))
   (:method ((object hash-table))
     (let ((new (make-hash-table :test 'equal)))
       (maphash (lambda (key val)
                  (setf (gethash key new) val))
                object)
       new))
-  (:documentation "Copy the OBJECT, potentially creating an identical one.
-Coerces all the encountered sequences to vectors."))
```

Comment:
Why remove this note on vectors?

Reply:
Because it's now vector -> adjustable vector transformation, which is more or less obvious. I've documented it in d4d20440afe5907d4a3074eb2ec9b427840757b1, though.